examples/cloudflare-workers/with-be-emailpassword/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "with-be-email-password-hono",
```

Comment:
Please rename this as `with-email-password-hono-be-only` (incl the directory)

Reply:
Done